lib/utilities/api_handlers/zuri_api.dart

```diff
@@ -119,6 +119,39 @@ class ZuriApi implements Api {
     }
   }
 
+  Future<dynamic> postDM(
```

Comment:
You're not overriding this function?

Besides you did not also include in the api.dart file

Reply:
It gives an error when it's overridden.

Reply:
The method doesn't override an inherited method

Reply:
Yes @Starhkz you'd have to add it to the api.dart file so that it inherits it from there

Reply:
Exactly